<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>nomadwindow</class>
 <widget class="QWidget" name="nomadwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>362</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::NoContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>Nomad Window</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="tabStatus">
      <attribute name="title">
       <string>Status</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Power System (V)</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutC">
            <item>
             <widget class="QLabel" name="labelCPU">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Computer </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineVCPU">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>50</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>--,--</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="barVCPU">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="minimum">
               <number>1080</number>
              </property>
              <property name="maximum">
               <number>1200</number>
              </property>
              <property name="value">
               <number>1080</number>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayoutM">
            <item>
             <widget class="QLabel" name="labelMotor">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Motor</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineVMotor">
              <property name="enabled">
               <bool>true</bool>
              </property>
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>50</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>50</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>50</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>--,--</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QProgressBar" name="barVMotor">
              <property name="styleSheet">
               <string/>
              </property>
              <property name="minimum">
               <number>2160</number>
              </property>
              <property name="maximum">
               <number>2400</number>
              </property>
              <property name="value">
               <number>2160</number>
              </property>
              <property name="textVisible">
               <bool>true</bool>
              </property>
              <property name="invertedAppearance">
               <bool>false</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>System Data</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <widget class="QLabel" name="labelVel">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Velocity</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineVel">
              <property name="text">
               <string>--,-- mm/s</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5">
            <item>
             <widget class="QLabel" name="labelTor">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Turret</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineTor">
              <property name="text">
               <string>--,-- deg/s</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6">
            <item>
             <widget class="QLabel" name="labelSte">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>0</height>
               </size>
              </property>
              <property name="text">
               <string>Steering</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="lineSte">
              <property name="text">
               <string>--,-- deg/s</string>
              </property>
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabCommands">
      <attribute name="title">
       <string>Commands</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QCheckBox" name="cBFixed">
         <property name="text">
          <string>Fixed steering and turret</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
         <property name="tristate">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_7">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Robot :</string>
           </property>
           <property name="buddy">
            <cstring>tabWidget</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="robotSim">
           <property name="text">
            <string>Simulado</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="robotReal">
           <property name="text">
            <string>Real</string>
           </property>
           <property name="checked">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pBTalk">
         <property name="text">
          <string>Talk</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pBStop">
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pBZR">
         <property name="text">
          <string>Zero Reset</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>lineVCPU</tabstop>
  <tabstop>lineVMotor</tabstop>
  <tabstop>lineVel</tabstop>
  <tabstop>lineTor</tabstop>
  <tabstop>lineSte</tabstop>
  <tabstop>cBFixed</tabstop>
  <tabstop>robotSim</tabstop>
  <tabstop>robotReal</tabstop>
  <tabstop>pBTalk</tabstop>
  <tabstop>pBStop</tabstop>
  <tabstop>pBZR</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pBTalk</sender>
   <signal>clicked()</signal>
   <receiver>nomadwindow</receiver>
   <slot>BotonTalk()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>285</x>
     <y>151</y>
    </hint>
    <hint type="destinationlabel">
     <x>294</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBStop</sender>
   <signal>clicked()</signal>
   <receiver>nomadwindow</receiver>
   <slot>BotonStop()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>222</x>
     <y>187</y>
    </hint>
    <hint type="destinationlabel">
     <x>293</x>
     <y>149</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pBZR</sender>
   <signal>clicked()</signal>
   <receiver>nomadwindow</receiver>
   <slot>BotonZR()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>223</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>182</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>robotSim</sender>
   <signal>toggled(bool)</signal>
   <receiver>nomadwindow</receiver>
   <slot>Simulado(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>180</x>
     <y>114</y>
    </hint>
    <hint type="destinationlabel">
     <x>292</x>
     <y>42</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cBFixed</sender>
   <signal>toggled(bool)</signal>
   <receiver>nomadwindow</receiver>
   <slot>FixedTurret(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>75</x>
     <y>66</y>
    </hint>
    <hint type="destinationlabel">
     <x>231</x>
     <y>6</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>BotonTalk()</slot>
  <slot>BotonStop()</slot>
  <slot>BotonZR()</slot>
  <slot>Simulado(bool)</slot>
  <slot>FixedTurret(bool)</slot>
 </slots>
</ui>
